docs/querying/sql.md

```diff
@@ -82,6 +83,27 @@ FROM clause, metadata tables are not considered datasources. They exist only in
 For more information about table, lookup, query, and join datasources, refer to the [Datasources](datasource.md)
 documentation.
 
+## UNNEST
+
+The UNNEST clause unnests values stored in arrays within a column. It's the SQL equivalent to the [unnest datasource](./datasource.md#unnest).
```

Comment:
unnest isn't limited to working on arrays in a column (we don't currently have any array typed columns). `MV_TO_ARRAY` effectively allows casting a multi-value string to an `ARRAY<STRING>`, but arrays could also come from other virtual columns, such as anything using the `ARRAY` constructor or anything that can build arrays, such as `ARRAY_AGG`